services/AuthService.js

```diff
@@ -143,7 +182,10 @@ class AuthService {
       expiresIn: jwtExpirySeconds
     })
     res.cookie("token", newToken, { maxAge: jwtExpirySeconds * 1000 });
-    res.end();
+    res.send({
+      token: newToken,
```

Comment:
Is there a reason for sending the token back in the body since it's already getting sent back as a cookie?